tv_apps/smart-home/js/card_picker.js

```diff
@@ -248,6 +250,18 @@
       function appendToGridView(appCardElem) {
         that.gridView.appendChild(appCardElem);
         that.appCardElems.push(appCardElem);
+
+        if (top >= 0) {
```

Comment:
The logic here is not descriptive enough, including the meaning of `top = -1` and variable name of `cr`.